packages/cashier/src/Containers/withdrawal.jsx

```diff
@@ -19,6 +20,11 @@ const WithdrawalSideNote = () => {
             i18n_default_text='Do not enter an address linked to an ICO purchase or crowdsale. If you do, the ICO tokens will not be credited into your account.'
             key={0}
         />,
+        <Localize
+            i18n_default_text='It may take up to <0>6 confirmations</0> for your funds to be reflected in your destination wallet.'
+            key={0}
```

Comment:
I don't think you need this though?

Reply:
I'm not sure if this key has a special purpose or if it's just to uniquely identify elements in a list. If it's the latter then I guess the key value should be 1 (since 0 is the key of the previous component in the list).